src/types/data.d.ts

```diff
@@ -0,0 +1,293 @@
+interface Dipatch {
```

Comment:
typo? Dispatch?

Reply:
Oops, yes. That is a typo.

Reply:
But this will only stay till the entire codebase has not migrated to TS. We actually don't need this dispatch type as the actions now can be incorporated into the component itself.